src/cnd/cnd.spec.ts

```diff
@@ -0,0 +1,102 @@
+import nock = require("nock");
+import { Problem } from "./axios_rfc7807_middleware";
+import { Cnd } from "./cnd";
+
+it.each([
+  ["500", "Internal Server Error", "about:blank", null],
+  ["405", "Invalid action invocation", null, null],
+  [
+    "400",
+    "Swap not supported",
```

Comment:
@thomaseizinger did you consider supporting a list of problems that cnd could return to allow the SDK consumer's to have better expectation and individual problem management?

Reply:
Considered it yes but the way I would go about it is to actually have those problems point to a documentation using the `type` URL. See the RFC for details: https://tools.ietf.org/html/rfc7807#section-3.1

Those URLs can then be used by clients to differentiate between different errors.

I would say the current state is fine. Creating like a hard-coded list of problems would be over-engineered IMO and just create a maintenance burden.